ert3/config/_stages_config.py

```diff
@@ -58,19 +59,21 @@ def _ensure_input_record_mime(cls, field: str, values: Dict[str, Any]) -> str:
 
 
 class Record(_MimeBase):
-    record: str
-    location: str
+    name: str = Field(None, alias="record")
+    path: pathlib.Path = Field(None, alias="location")
```

Comment:
One reason why this was _location_ and not _path_ is due to function steps. A `Record` may never touch the disk in that case, but rather a variable in memory. So I think this won't work.

Reply:
All aliases are removed to reduce/remove confusion.